### PR TITLE
Patient list Reports for MER Indicators

### PR DESCRIPTION
**Pull request by ssmusoke**
https://app.asana.com/0/1131715315801874/1200590245301100

1. Added Patient Lists for the following reports and verified them with the results 
- TX_NEW
- TX_CUR 28 and 90 days
- TX_TB
- TB_ART
- TX_ML
2. Deleted the pre-2019 versions of the following reports 
- TX_Current
- TX_New
- TX_TB

3.  Removed references to addIndicator and addGender functions from the SetupTXNewReport to a Helper function

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTxCurrent_28Days2019ReportPatientListReport.java [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+        rd.setName(getName());
+        rd.setDescription(getDescription());
+        rd.setParameters(getParameters());
+
+        PatientDataSetDefinition dsd = new PatientDataSetDefinition();
+
+        dsd.setParameters(getParameters());
+        rd.addDataSetDefinition("TX_CUR_PatientList", Mapped.mapStraightThrough(dsd));
+
+        CohortDefinition returnToCareClients= hivCohortDefinitionLibrary.getActivePatientsWithLostToFollowUpAsByDays("28");
```
Comment (dbaluku) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTxCurrent_28Days2019ReportPatientListReport.java:128]: Rename the cohort definition to a proper name according to the report

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTxCurrent_90Days2019ReportPatientListReport.java [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        rd.setName(getName());
+        rd.setDescription(getDescription());
+        rd.setParameters(getParameters());
+
+        PatientDataSetDefinition dsd = new PatientDataSetDefinition();
+
+        dsd.setParameters(getParameters());
+        rd.addDataSetDefinition("TX_CUR_PatientList", Mapped.mapStraightThrough(dsd));
+
+        CohortDefinition returnToCareClients= hivCohortDefinitionLibrary.getActivePatientsWithLostToFollowUpAsByDays("90");
```
Comment (dbaluku) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTxCurrent_90Days2019ReportPatientListReport.java:128]: Rename the cohort Definition to TX_CURR

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports2019/SetupHCT_TST_Facility2019Report.java [hunk 3]
```diff
@@ -189,15 +186,15 @@ public ReportDefinition constructReportDefinition() {
...
         addAgeAndGender(dsd,"t","tested negative through the Other PITC Clinic",df.getPatientsInAll(patientThroughOtherEntryPoints,patientsTestedThroughVCTApproach,testedNegativeDuringPeriod));
 
-        setupTxNewReport.addIndicator(dsd,"PIPa","PIPa positive females",df.getPatientsInAll(females,PIPS,testedPositiveDuringPeriod,patientsTestedThroughHealthFacility),"");
-        setupTxNewReport.addIndicator(dsd,"PIPb","PIPb positive males",df.getPatientsInAll(males,PIPS,testedPositiveDuringPeriod,patientsTestedThroughHealthFacility),"");
-        setupTxNewReport.addIndicator(dsd,"PIPc","PIPc negative females",df.getPatientsInAll(females,PIPS,testedNegativeDuringPeriod,patientsTestedThroughHealthFacility),"");
-        setupTxNewReport.addIndicator(dsd,"PIPd","PIPd negative males",df.getPatientsInAll(males,PIPS,testedNegativeDuringPeriod,patientsTestedThroughHealthFacility),"");
+        org.openmrs.module.ugandaemrreports.reports.Helper.addIndicator(dsd,"PIPa","PIPa positive females",df.getPatientsInAll(females,PIPS,testedPositiveDuringPeriod,patientsTestedThroughHealthFacility),"");
```
Comment (dbaluku) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/reports2019/SetupHCT_TST_Facility2019Report.java:189]: I think this  should be just 'Helper' without the package path

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTxCurrent_90Days2019ReportPatientListReport.java (hunk 2 above)
Comment (ssmusoke) [reply api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTxCurrent_90Days2019ReportPatientListReport.java]: @dbaluku Done

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTxCurrent_28Days2019ReportPatientListReport.java (hunk 1 above)
Comment (ssmusoke) [reply api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTxCurrent_28Days2019ReportPatientListReport.java]: @dbaluku Done

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports2019/SetupHCT_TST_Facility2019Report.java (hunk 3 above)
Comment (ssmusoke) [reply api/src/main/java/org/openmrs/module/ugandaemrreports/reports2019/SetupHCT_TST_Facility2019Report.java]: @dbaluku Done

**Pull request merged**